### DISCUSSION TITLE
Change language

### DISCUSSION THREAD

**Discussion by JavGuerra** [daattali/beautiful-jekyll#973] [Q&A] [answered]
Hello, thank you for this theme. I just created my blog on github with it. My posts are going to be in Spanish, but I can't get the site in that language to show the buttons in Spanish. I tried putting "languaje: es" in "_config.yml" but it didn't work. I have seen that there is a file in "data/ui-text.yml" with the translations, How to use it? Greetings.

**Comment by daattali** [comment, marked as answer]
That's for staticman comments only, not for the whole site

**Reply by JavGuerra** [reply]
Thank you. I will try to make the changes to the files manually.
